pandas/tests/series/indexing/test_indexing.py

```diff
@@ -102,8 +101,8 @@ def test_getitem_setitem_ellipsis():
 )
 def test_getitem_with_duplicates_indices(result_1, duplicate_item, expected_1):
     # GH 17610
-    result = result_1.append(duplicate_item)
-    expected = expected_1.append(duplicate_item)
+    result = result_1._append(duplicate_item)
```

Comment:
Do you think it might be possible/worth replacing internal usages of `_append` so we can remove it?

Reply:
IIRC we kept _append in part bc it is used in a handful of places internally

Reply:
Gotcha. Maybe it can be investigated independently if those internal usages of `_append` could be replace with another op in the future